src/main/scala/org/broadinstitute/dsde/firecloud/EntityService.scala

```diff
@@ -368,29 +326,20 @@ class EntityClient(requestContext: RequestContext, modelSchema: ModelSchema, goo
   }
 
   def importPFB(workspaceNamespace: String, workspaceName: String, pfbRequest: PfbImportRequest, userInfo: UserInfo): Future[PerRequestMessage] = {
+    importServiceDAO.importPFB(workspaceNamespace, workspaceName, pfbRequest)(userInfo)
+  }
 
-    // the payload to Import Service sends "path" and filetype.  Here, we force-hardcode filetype because this API
-    // should only be used for PFBs.
-    val importServicePayload: ImportServiceRequest = ImportServiceRequest(path = pfbRequest.url.getOrElse(""), filetype = "pfb")
-
-    val importServiceUrl = FireCloudDirectiveUtils.encodeUri(s"${FireCloudConfig.ImportService.server}/$workspaceNamespace/$workspaceName/imports")
-
-    userAuthedRequest(Post(importServiceUrl, importServicePayload))(userInfo) map {
-      case resp if resp.status == Created =>
-        val importServiceResponse = unmarshal[ImportServiceResponse].apply(resp)
-        // for backwards compatibility, we return Accepted(202), even though import service returns Created(201),
-        // and we return a different response payload than what import service returns.
-
-        val responsePayload:PfbImportResponse = PfbImportResponse(
-          jobId = importServiceResponse.jobId,
-          url = pfbRequest.url.getOrElse(""),
-          workspace = WorkspaceName(workspaceNamespace, workspaceName)
-        )
+  def getEntitiesWithType(workspaceNamespace: String, workspaceName: String, userInfo: UserInfo): Future[PerRequestMessage] = {
```

Comment:
this used to live in `GetEntitiesWithType.scala`. i've moved it here and greatly simplified the code